runtime-util/src/main/java/io/camunda/connector/runtime/util/feel/FeelEngineWrapperException.java

```diff
@@ -18,6 +18,14 @@
 
 public class FeelEngineWrapperException extends RuntimeException {
 
+  public FeelEngineWrapperException(
+      final String reason, final String expression, final Object context) {
+    super(
+        String.format(
+            "Failed to evaluate expression '%s' and context '%s', because %s",
+            expression, context, reason));
```

Comment:
Let's avoid code duplication.
```suggestion
    this(reason, expression, context, null);
```